rma/models/rma.py

```diff
@@ -528,10 +528,23 @@ def unlink(self):
                 _("You cannot delete RMAs that are not in draft state"))
         return super().unlink()
 
+    def _send_confirmation_email(self):
+        """Auto send notifications"""
+        for rma in self.filtered(lambda p: p.company_id.send_rma_confirmation):
+            rma_template_id = (
+                rma.company_id.rma_mail_confirmation_template_id.id
+            )
+            rma.with_context(
+                force_send=True, 
+                mark_rma_as_sent=True
```

Comment:
Why this context?

Reply:
It changes the state of the RMA `sent` field when it passes by `message_post`: https://github.com/OCA/rma/blob/12.0/rma/models/rma.py#L1144